apps/courses/src/routes/index.tsx

```diff
@@ -37,6 +39,8 @@ export default () => (
       <Route path="/" element={<CoursesSearch />} />
       <Route path=":slug" element={<CourseOverview />} />
     </Route>
+    <Route path="policy" element={<PrivacyPolicy />} />
+    <Route path="terms" element={<TermsOfService />} />
```

Comment:
This page is completely identical to the privacy policy. The only difference is the content.

THEREFORE - you should delete the terms of service folder entirely and rename the "privacy-policy" folder to be "policy-and-terms". You can determine the content of the page by looking at the URL when the component loads and then loading the appropriate copy. For the routes file here, you will have two routes, each with the exact same element and the content determined inside the component.